Pipeline/RunIDZCodeReview/codereview.properties

```diff
@@ -21,7 +22,6 @@ codereview_includedFiles=**/*.cbl
 ## IDZ code review supports COBOL and PLI, Sample:codereview_includedFiles=**/*.cbl
```

Comment:
Can you just briefly update the comment for the variable.

--> This is used to identify the PDS libraries for the Copybooks based on the file extension of the include files

Reply:
Done